addon/components/o-s-s/progress-bar.ts

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+enum ProgressBarSkins {
+  'primary',
+  'warning',
+  'success'
+}
+
+interface OSSProgressBarArgs {
+  value: number;
+  label?: string;
+  valueIsVisible?: boolean;
+  skin?: ProgressBarSkins;
+  small?: boolean;
+}
+
+export default class OSSProgressBar extends Component<OSSProgressBarArgs> {
+  constructor(owner: unknown, args: OSSProgressBarArgs) {
+    super(owner, args);
+
+    assert(
+      '[component][OSS::ProgressBar] You must pass a numbered value between 0 and 100',
+      0 <= args.value && args.value <= 100
```

Comment:
```suggestion
      '[component][OSS::ProgressBar] You must pass a numbered value between 0 and 100',
      0 >= args.value && args.value <= 100
```
The range is not good here, negative value pass you range here

Reply:
Sorted that one out too